docs/usage.md

```diff
@@ -12,49 +12,85 @@ The input data can be passed to nf-core/mag in two possible ways, either using t
 
+If short reads are provided (`short_reads_1`), the `short_reads_platform` column is required. Valid options include:
+
+- `ILLUMINA`, `BGISEQ`, `LS454`, `ION_TORRENT`, `DNBSEQ`, `ELEMENT`, `ULTIMA`, `VELA_DIAGNOSTICS`, `GENAPSYS`, `GENEMIND`, `TAPESTRI`.
+
+If long reads are provided (`long_reads`), the `long_reads_platform` column is required. Valid options include:
+
+- `OXFORD_NANOPORE`: Oxford Nanopore Technologies (ONT) reads, which may have higher error rates compared to newer technologies.
+- `OXFORD_NANOPORE_HQ`: High-quality ONT reads, typically with an error rate of less than 5%, achievable with the latest ONT chemistries and sequencing platforms. This option should generally be used unless working with older data.
+- `PACBIO_SMRT`: Pacific Biosciences Single Molecule Real-Time (SMRT) sequencing reads.
+
+These platform fields are important for downstream alignment and assembly tools.
+
 At a minimum CSV file should contain the following columns:
 
-`sample,group,short_reads_1,short_reads_2,long_reads`
+`sample,group,short_reads_1,short_reads_2,long_reads,short_reads_platform,long_reads_platform`
 
-The path to `long_reads` and `short_reads_2` is optional. Valid examples could look like the following:
+In cases of short-read only data run, the path to `long_reads` and `short_reads_2` is optional. Valid examples could look like the following:
 
-```csv title="samplesheet.csv"
-sample,group,short_reads_1,short_reads_2,long_reads
-sample1,0,data/sample1_R1.fastq.gz,data/sample1_R2.fastq.gz,data/sample1.fastq.gz
-sample2,0,data/sample2_R1.fastq.gz,data/sample2_R2.fastq.gz,data/sample2.fastq.gz
-sample3,1,data/sample3_R1.fastq.gz,data/sample3_R2.fastq.gz,
+```csv title="samplesheet_mix.csv"
+sample,group,short_reads_1,short_reads_2,long_reads,short_reads_platform,long_reads_platform
+sample1,0,data/sample1_R1.fastq.gz,data/sample1_R2.fastq.gz,data/sample1.fastq.gz,ILLUMINA,OXFORD_NANOPORE
+sample2,0,data/sample2_R1.fastq.gz,data/sample2_R2.fastq.gz,data/sample2.fastq.gz,ILLUMINA,OXFORD_NANOPORE
+sample3,1,data/sample3_R1.fastq.gz,data/sample3_R2.fastq.gz,,ILLUMINA,
 ```
 
 or
 
-```csv title="samplesheet.csv"
-sample,group,short_reads_1,short_reads_2,long_reads
-sample1,0,data/sample1.fastq.gz,,
-sample2,0,data/sample2.fastq.gz,,
+```csv title="samplesheet_shortreadonly.csv"
+sample,group,short_reads_1,short_reads_2,long_reads,short_reads_platform
+sample1,0,data/sample1.fastq.gz,,,ILLUMINA
+sample2,0,data/sample2.fastq.gz,,,ILLUMINA
 ```
 
 or to additionally to perform run merging of two runs of sample1:
```

Comment:
There's one "to" too many, but I don't know which should go.

Reply:
OK will fix!